webapp/src/cache/lfu_cache.js

```diff
@@ -0,0 +1,61 @@
+
+export default class LFUCache {
+    constructor(size) {
+        this.size = size;
+        this.data = {};
+    }
+
+    /**
+     * @param {string} key, postID for which we need a message
```

Comment:
Class is general and can be used for other data, so I think comments are too specific